docs/docs/technology/mantis.md

```diff
@@ -1,40 +1,33 @@
 
-There is no one set definition for this Intents in DeFi. In general, intents are understood to be users’ desires for a given transaction or other outcome. Intents include desired parameters (such as to swap X amount of A token for B token), but leave some room for flexibility (such as where this swap occurs) in the solution that solvers provide. Anoma’s Apriori does a great job of breaking down the history and various definitions of intents in this blog about their Intents Day event. This blog by Essential also provides another means of defining intents and solvers.
+Presently, MANTIS is live on mainnet for testing with a number of solvers to fulfill user intents, with the inclusion of oracles and collators to run validators for this framework.
 
-Composable believes that:
+:::info Understanding Intents
 
-- Intents are inherently cross-domain
-- Blockchains are inherently trust-no-one markets
+Intents have become a hot topic in DeFi, though there is no one set definition for this concept. In general, intents are understood to be users’ desires for a given transaction or other outcome. Intents include desired parameters (such as to swap X amount of A token for B token), but leave some room for flexibility (such as where this swap occurs) in the solution that solvers provide. Anoma does a great job of breaking down the history and various definitions of intents in [this blog](https://anoma.net/blog/intents-arent-real) about their Intents Day event. [This blog](https://blog.essential.builders/introducing-essential/) by Essential also provides another means of defining intents and solvers.
+:::
+## Intents are Inherently Cross-Domain
 
-Distinct blockchain ecosystems, though increasing the solution space for problems, do not actually communicate with each other. There is thus a huge amount of potential loss due to lack of synchronicity between these markets. There is no credibly verifiable way to settle orders between chains, and also settle them at the same time. 
+Composable's thesis introduces an additional feature to how intents are currently recognised, interoperability - intents are inherently cross-domain. To understand this, the history of the equities markets provides great insights as it became more efficient with execution being broken across different markets. Prior to the introduction of dark pools, which are private liquidity pools, individuals were only able to use select markets:
 
-So, how can the supply chain be optimised so that there is the remote possibility of facilitating this?
+After the introduction of dark pools, which enabled orders to be broken up and split into different markets, the solution space broadened and more earnings opportunities were made available:
```

Comment:
in blockchain we call privacy, mev resistance and security, not dark? like if dark is bad? :)

Reply:
this is referring to dark pools in traditional stock markets, basically private exchanges used for securities trading.